src/javascript_bindings.pyx

```diff
@@ -29,14 +29,15 @@ cdef class JavascriptBindings:
     cpdef py_void SetFunction(self, py_string name, object func):
         self.SetProperty(name, func)
 
-    cpdef py_void SetObject(self, py_string name, object obj):
+    cpdef py_void SetObject(self, py_string name, object obj,
+                            object allow_properties=False):
         if not hasattr(obj, "__class__"):
             raise Exception("JavascriptBindings.SetObject() failed: name=%s, "
                             "__class__ attribute missing, this is not an object" % name)
         cdef dict methods = {}
         cdef py_string key
         cdef object method
-        cdef object predicate = inspect.ismethod
+        cdef object predicate = None if allow_properties else inspect.ismethod
```

Comment:
Using "False if.." would be better, not a big deal, but just made me think if None has some special meaning